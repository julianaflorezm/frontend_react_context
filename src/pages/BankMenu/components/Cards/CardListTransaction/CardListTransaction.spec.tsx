import { render } from '@testing-library/react';
import { TransactionCardListProps, CardListTransaction } from './index';

describe('CardListTransaction Test', () => {
  let props: TransactionCardListProps;

  beforeEach(() => {
    props = {
      transactions: [
        {
          id: 1,
          valor: 1000,
          costo: 0,
          esCuentaOrigen: false,
          fechaCreacion: new Date('2022-03-02 11:00:00'),
          origen: {
            id: 1,
            nombre: 'Cuenta de ahorro',
            numeroCuenta: '234vf5',
            saldo: 50000,
            fechaCreacion: '2022-03-05 11:00:00',
          },
          destino: {
            id: 2,
            nombre: 'Cuenta de ahorro',
            numeroCuenta: 'fgt65yg',
            saldo: 50000,
            fechaCreacion: '2022-03-04 11:00:00',
          },
        },
      ],
    };
  });

  it('should match snapshot', () => {
    const { container } = render(<CardListTransaction {...props} />);
    expect(container).toMatchSnapshot();
  });
});
