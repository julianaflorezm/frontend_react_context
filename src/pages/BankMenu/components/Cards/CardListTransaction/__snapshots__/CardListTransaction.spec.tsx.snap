// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardListTransaction Test should match snapshot 1`] = `
<div>
  <div
    class="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
  >
    <div
      class="MuiCardContent-root"
    >
      <p
        class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
      >
        2 de marzo de 2022
      </p>
      <p
        class="MuiTypography-root MuiTypography-body1"
      >
        Transferencia desde 
        Cuenta de ahorro
         
      </p>
      <p
        class="MuiTypography-root makeStyles-destino-2 MuiTypography-body2"
      >
        $ 
        1000
      </p>
    </div>
  </div>
</div>
`;
